README.md

```diff
@@ -1,9 +1,40 @@
+
 # Minionify
-An app that allows basic control over remote devices.
 
+## Introduction
+An app that allows basic control over remote devices.
 The users would be divided into 2 groups; managers and minions.
 A manager can use the app to execute certain operations on the minions' devices, such as gathering statistics, sending notifications, determining location, opening another app etc.
-
 The minions would use the app passively; by using the app, they willingly give the managers permission to execute operations on their devices.
+Each minion can have only one manager, while managers can have multiple minions.
 
 Possible users of the app are worried parents with their children or strict bosses with their employees.
+
+
+## Modes
+The Minionify app will have two different modes that it can operate:
+
+- As A Minion: Minionify will run a background service that will listen for incoming requests from the managers, execute them and send back the response back to the manager.
+- As A Manager: Minionify will display a GUI to the user that will let them control their minions apps.
+For example, a user can choose one of its minions and get their location, statistics and more...
+
+## Protocol
+The Minionify app will use Firebase as its server. 
```

Comment:
i don't know if server is the correct word, as this is not exactly a server-client design. maybe "for communication between the minions and the managers"

Reply:
Done